src/app/docs/kagent/crds/crds.mdx

```diff
@@ -0,0 +1,140 @@
+# Kagent Custom Resource Definitions (CRDs)
+
+## What are CRDs?
+
+Custom Resource Definitions (CRDs) in Kubernetes provide a mechanism to extend the Kubernetes API by defining new, custom resource types. These custom resources behave similarly to built-in Kubernetes objects like **Pods** or **Deployments**, but are tailored to specific application or organizational needs.
+
+---
+
+### Installing Kagent CRDs
+
+> ⚠️ Skip this step if already done in **Quick Start**.
+
+```bash
+kagent install
+```
+To access the list of CRDs:
+```bash
+kubectl get crd -n kagent
+```
+### List of CRDs
+1. agents.kagent.dev
+2. mcpservers.kagent.dev
+3. memories.kagent.dev
+4. modelconfigs.kagent.dev
+5. remotemcpservers.kagent.dev
+6. teams.kagent.dev
+7. toolservers.kagent.dev
+
+### You can describe any CRD using
+```bash
+kubectl describe crd "name-of-crd" -o yaml
+```
+### CRD Definitions
+1. agents.kagent.dev
+```yaml
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: agents.kagent.dev
+spec:
+  group: kagent.dev
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          properties:
+            spec:
+              properties:
+                replicas:
+                  type: integer
+                image:
+                  type: string
+                resources:
+                  type: object
+  scope: Namespaced
+  names:
+    plural: agents
+    kind: Agent
+```
+2. mcpservers.kagent.dev
+```yaml
+apiVersion: kagent.dev/v1alpha1
+kind: MCPServer
+metadata:
+  name: example-mcp
+spec:
+  endpoint: "http://mcp.example.com"
```

Comment:
this is incorrect -- https://github.com/kagent-dev/kmcp/blob/41071d12350f22dd20492f5848e0629ba5ee7c0c/api/v1alpha1/mcpserver_types.go#L138